configure.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
```

Comment:
If you provide a docker image with your repo, i would assume the defult way of working is to use the povied docker. If someone wants to use a "local" setup it is on him to figure out how to do it :)

I would say this script is not needed